### DISCUSSION TITLE
Kenyas country code is missing when setting up 2FA

### DISCUSSION THREAD

**Discussion by AzengaKevin** [community/community#89120] [Other Feature Feedback, Questions, & Ideas]
Can you please add +254 to the **Country or region code** control.

**Comment by ghostinhershell** [comment]
Hi @AzengaKevin 👋🏾 

Thank so much for writing in. If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

You can also find this, and other frequently asked questions, in our [Two Factor Authentication Frequently Asked Questions 🔑 🗝️](https://github.com/orgs/community/discussions/68240) post.

Let me know if you have any additional questions. Thanks!
